platforms/hyperledger-indy/charts/indy-auth-job/templates/job.yaml

```diff
@@ -12,6 +12,7 @@ spec:
       restartPolicy: OnFailure
       imagePullSecrets:
         - name: "{{ $.Values.image.pullSecret }}"
+      when: (network.docker.username | default('', true) | trim != '') and (network.docker.password | default('', true) | trim != '')  
```

Comment:
Dont think you have changed anything here?

Reply:
I had replaced && with and. Can you please guide me with the exact changes needed to be done in the job.yml file?

Reply:
check `platforms/hyperledger-indy/charts/indy-node/templates/service.yaml` on how if condition is defined in a helm chart. `when` is wrong, you can check your helm charts by running `helm lint`